### DISCUSSION TITLE
Is Dependabot able to scan Docker/OCI images?

### DISCUSSION THREAD

**Discussion by 53845714nF** [community/community#128902] [Code Security] [answered]
### Select Topic Area

Question

### Body

I have just read [GitHub Dependabot docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file?learn=dependency_version_updates&learnProduct=code-security#docker)

If I understand correctly, Dependabot is able to scan my repo, including my Dockerfile.
It is also able to log in to my registry to make updates. 

But does it also look directly into the images? It could be that my base image is compromised.

**Comment by SajanGhimire1** [comment, marked as answer]
@53845714nF 
No, Dependabot does not scan Docker or OCI images directly. It only updates the base images specified in your Dockerfile. For scanning Docker images for vulnerabilities, use GitHub Advanced Security's container scanning features which identifies & reports vulnerabilities in your images.

**Reply by rdesai3** [reply]
@54J4N , could you provide documentation for GitHub Advanced Security's container scanning features? I cannot find container scanning feature in GHAS.

**Comment by 53845714nF** [comment]
Thank you for your answer.
